queries.sql

```diff
@@ -35,4 +35,61 @@ SELECT COUNT(*) FROM animals WHERE escape_attempt = 0;
 SELECT AVG(weight_kg) FROM animals;
 SELECT MAX(escape_attempt) FROM animals;
 SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
-SELECT AVG(escape_attempt) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
\ No newline at end of file
+SELECT AVG(escape_attempt) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
+
+
+-- Link animal table to species table and owners table
+
+-- Update species_id in animal table
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+-- Update owner_id in animals table
+UPDATE animals SET owners_id = 1 WHERE name = 'Agumon';
+
+UPDATE animals SET owners_id = 2 WHERE name  = 'Gabumon' OR name ='Pikachu';
+
+UPDATE animals SET owners_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+
+UPDATE animals SET owners_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+
+UPDATE animals SET owners_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
```

Comment:
### **Optional**
- [ ] Please remove all unnecessary code from the `queries.sql` file

Reply:
okay senior